<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainScreen</class>
 <widget class="QWidget" name="mainScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1099</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fff;</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>851</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="handleWidth">
    <number>0</number>
   </property>
   <widget class="QWidget" name="menu_widget" native="true">
    <property name="minimumSize">
     <size>
      <width>278</width>
      <height>850</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>292</width>
      <height>851</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #06162d;
color: #fff;
border: none;</string>
    </property>
    <widget class="QToolBox" name="toolBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>241</width>
       <height>571</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#toolBox{
	color: #fff;
}

#toolBox::tab{
	padding-left: 5px;
	text-align: left;
	border-radius: 2px;
}

#toolBox::tab:selected{
	background-color: #2d9cdb;
	font-weight: bold;
}

#toolbox QPushButton{
	padding: 5px 0px 5px 20px;
	text-align: left;
	border-radius: 3px;
}

#toolBox QPushButton::hover{
	background-color: #85C1E9;
}
#toolBox QPushButton::checked{
	background-color: #4398D8;
}</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>241</width>
        <height>463</height>
       </rect>
      </property>
      <attribute name="label">
       <string>PÃ¡gina</string>
      </attribute>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Account</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>241</width>
        <height>463</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/iconos/project-128.ico</normaloff>:/iconos/project-128.ico</iconset>
      </attribute>
      <attribute name="label">
       <string>Proyecto</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QPushButton" name="verProyectoButton">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Ver Proyecto</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/iconos/project-128.ico</normaloff>:/iconos/project-128.ico</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="crearProyectoButton">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Crear Proyecto</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/iconos/edit-property-128.ico</normaloff>:/iconos/edit-property-128.ico</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="eliminarProyectoButton">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Eliminar Proyecto</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/iconos/trash-9-128.ico</normaloff>:/iconos/trash-9-128.ico</iconset>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>393</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>241</width>
        <height>463</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/iconos/check-mark-2-128.ico</normaloff>:/iconos/check-mark-2-128.ico</iconset>
      </attribute>
      <attribute name="label">
       <string>To Do</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QPushButton" name="toDoButton">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>To Do</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/iconos/check-mark-2-128.ico</normaloff>:/iconos/check-mark-2-128.ico</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>455</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="main_widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QWidget" name="search_widget" native="true">
       <property name="styleSheet">
        <string notr="true">#search_widget{
	background-color: #ABB2B9;
}</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="menu_Button">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/iconos/menu-4-16.ico</normaloff>:/iconos/menu-4-16.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>138</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="user_label">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#user_label{
	background-color: #fff;
	border: 1px solid #F2F4F4;
	padding: 5px 5px;
	border-radius: 15%;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="res.qrc">:/iconos/contacts-128.ico</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>PÃ¡gina</string>
        </attribute>
        <widget class="QWidget" name="widget" native="true">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>751</width>
           <height>381</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>751</width>
           <height>381</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #06162d;
color: #fff;</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>50</number>
          </property>
          <property name="topMargin">
           <number>50</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>50</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>470</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Apellido: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="inputApellido">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>InputApellido</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>470</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Nombre: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="inputNombre">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>InputNombre</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="minimumSize">
               <size>
                <width>470</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>DNI: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="inputDNI">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>InputDNI</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="minimumSize">
               <size>
                <width>470</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Correo: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="inputCorreo">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>InputCorreo</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>menu_Button</sender>
   <signal>toggled(bool)</signal>
   <receiver>menu_widget</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
